run_csm.py

```diff
@@ -0,0 +1,122 @@
+            "come out. So like everyone, when they come into the park, they get like this little "
+            "bracelet and then you can go punching question blocks around."
+        ),
+        "audio": "prompts/conversational_b.wav",
+    }
+}
+
+def load_prompt_audio(audio_path: str, target_sample_rate: int) -> torch.Tensor:
+    audio_tensor, sample_rate = torchaudio.load(audio_path)
+    audio_tensor = audio_tensor.squeeze(0)
+    if sample_rate != target_sample_rate:
+        audio_tensor = torchaudio.functional.resample(
+            audio_tensor, orig_freq=sample_rate, new_freq=target_sample_rate
+        )
+    return audio_tensor
+
+def prepare_prompt(text: str, speaker: int, audio_path: str, sample_rate: int) -> Segment:
+    audio_tensor = load_prompt_audio(audio_path, sample_rate)
+    return Segment(text=text, speaker=speaker, audio=audio_tensor)
+
+def main():
+    # Select the best available device, skipping MPS due to float64 limitations
+    if torch.cuda.is_available():
+        device = "cuda"
+    else:
+        device = "cpu"
+    print(f"Using device: {device}")
+    
+    try:
+        # Load model
+        model_path = hf_hub_download(repo_id="sesame/csm-1b", filename="ckpt.pt")
+        generator = load_csm_1b(model_path, device)
+        
+        # Prepare prompts
+        prompt_a = prepare_prompt(
+            SPEAKER_PROMPTS["conversational_a"]["text"],
+            0,
+            SPEAKER_PROMPTS["conversational_a"]["audio"],
+            generator.sample_rate
+        )
+        
+        prompt_b = prepare_prompt(
+            SPEAKER_PROMPTS["conversational_b"]["text"],
+            1,
+            SPEAKER_PROMPTS["conversational_b"]["audio"],
+            generator.sample_rate
+        )
+        
+        # Generate conversation
+        conversation = [
+            ("Hey how are you doing?", 0),
+            ("Pretty good, pretty good. How about you?", 1),
+            ("I'm great! So happy to be speaking with you today.", 0),
+            ("Me too! This is some cool stuff, isn't it?", 1)
+        ]
+        
+        # Generate each utterance
+        generated_segments = []
+        prompt_segments = [prompt_a, prompt_b]
+        
+        for text, speaker_id in conversation:
+            print(f"Generating: {text}")
+            audio_tensor = generator.generate(
+                text=text,
+                speaker=speaker_id,
+                context=prompt_segments + generated_segments,
+                max_audio_length_ms=10_000,
+            )
+            generated_segments.append(Segment(text=text, speaker=speaker_id, audio=audio_tensor))
+            
+            # Save individual utterance
+            filename = f"output_{len(generated_segments)}.wav"
+            torchaudio.save(
+                filename, 
+                audio_tensor.unsqueeze(0).cpu(), 
+                generator.sample_rate
+            )
+            print(f"Saved {filename}")
```

Comment:
I'd like to keep the script simple and minimal, let's remove this. Others can add specific functionality in forks if they need